core/dbt/include/global_project/macros/schema_tests/accepted_values.sql

```diff
@@ -11,32 +7,27 @@ with all_values as (
 
     from {{ model }}
 
-),
-
-validation_errors as (
-
-    select
-        value_field
-
-    from all_values
-    where value_field not in (
-        {% for value in values -%}
-            {% if quote_values -%}
-            '{{ value }}'
-            {%- else -%}
-            {{ value }}
-            {%- endif -%}
-            {%- if not loop.last -%},{%- endif %}
-        {%- endfor %}
-    )
 )
 
-select *
-from validation_errors
+select
+    {{ column_name }} as value_field
+
+from all_values
+where {{ column_name }} not in (
```

Comment:
Sorry! The previous suggestion I gave you was... exactly wrong 🤦  I missed the CTE above was stilly above. Silly. This is what I get for trying to review PRs in the middle of the day.

I reran the failing tests locally, and I'm _pretty_ sure that switching this one back will fix the failing tests:
```suggestion
    value_field

from all_values
where value_field not in (
```